devel/ansible/roles/dev/files/.bashrc

```diff
@@ -0,0 +1,6 @@
+# .bashrc
+
+export FLASK_APP=/vagrant/securitas/app.py
+export SECURITAS_CONFIG_PATH=/home/vagrant/securitas.cfg
+
+alias securitas-devel="poetry run flask run --host=0.0.0.0 -h0"
```

Comment:
the two flags here do the same thing, only one of them is needed.

Reply:
oh, *facepalm* thanks @relrod